### ISSUE TITLE
All textfield examples should be labelled

### ISSUE THREAD

**Issue by yalagand** [adobe/spectrum-web-components#742]
Please provide examples with visible labels that are associated with the textfield component. 
URL tested: https://opensource.adobe.com/spectrum-web-components/components/textfield

**Comment by Westbrook** [comment]
related to #543 and #475
